airflow/lineage/__init__.py

```diff
@@ -45,6 +48,20 @@ class Metadata:
     data: Dict = attr.ib()
 
 
+def _get_backend() -> Optional[LineageBackend]:
+    _backend: Optional[LineageBackend] = None
+
+    try:
+        _backend_str = conf.get("lineage", "backend")
+        _backend = import_string(_backend_str)  # pylint: disable=protected-access
```

Comment:
Should we add check that the backend is a sub class of `LineageBackend` as we do in `XComBackend`? Currently there's no common methods but in future this may help us with catching errors. WDYT?

Reply:
Sounds reasonable, I'll add it.